pendulum_control/include/pendulum_control/rtt_executor.hpp

```diff
@@ -103,12 +90,11 @@ class RttExecutor : public rclcpp::Executor
     // Cast the argument so that we can access the executor's state.
     RttExecutor * executor = static_cast<RttExecutor *>(arg);
     // If the input pointer was NULL or invalid, or if rclcpp has stopped, signal rttest to stop.
-    if (!executor || !rclcpp::ok()) {
```

Comment:
I think we can change the static method to receive no arguments. That would make it nicer for everyone, including the ROS Team. In this case, we need to split the PR in two parts. We will discuss this tomorrow.

Reply:
Done